crypto/kdf/sskdf.c

```diff
@@ -70,6 +70,7 @@ static const unsigned char kmac_custom_str[] = { 0x4B, 0x44, 0x46 };
 static int SSKDF_hash_kdm(const EVP_MD *kdf_md,
                           const unsigned char *z, size_t z_len,
                           const unsigned char *info, size_t info_len,
+                          unsigned int append_ctr,
```

Comment:
Perhaps the comment prior to this function should be updated to additionally refer to X9.63 and explain the `append_ctr` arg